.github/workflows/send-calypso-app-build-trigger.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+trigger_payload=`cat $GITHUB_EVENT_PATH`
+
+workflow_data="{
+	\"action\": \"$GITHUB_ACTION\",
+	\"actor\": \"$GITHUB_ACTOR\",
+	\"run_id\": \"$GITHUB_RUN_ID\",
+	\"run_num\": \"$GITHUB_RUN_NUMBER\",
+	\"repo\": \"$GITHUB_REPOSITORY\",
+	\"trigger_payload\": $trigger_payload
```

Comment:
Will this contain the PR number? I'm asking because we'll need it to post a comment with the diff link back to GitHub.

Reply:
Yes, it does. The information here is basically the same as the payload that would be sent for a Jetpack fusion request.